main.nf

```diff
@@ -946,9 +948,15 @@ process merge_featureCounts {
     file 'merged_gene_counts.txt'
 
     script:
+    if (input_files.size() == 1) {
+    """
+    mv $input_files[0] 'merged_gene_counts.txt'
+    """    
+    } else {
     """
-    merge_featurecounts.py -o merged_gene_counts.txt -i $input_files
+    csvtk join -t -f "Geneid,Start,Length,End,Chr,Strand,gene_name" $input_files |csvtk cut -t -f "-Start,-Chr,-End,-Length,-Strand" | sed 's/Aligned.sortedByCoord.out.markDups.bam//g' > merged_gene_counts.txt
```

Comment:
Replacing an entire script with a one-liner? Nice! @Hammarn will be jealous 😉 

Might be nice to split this out onto a few lines so that it's easier to read...

Reply:
Done !